### ISSUE TITLE
WebP thumbnails not showing on Windows File Explorer

### ISSUE THREAD

**Issue by riomccloud** [nomacs/nomacs#1020]
**Describe the bug**
Windows File Explorer shows all image file thumbnails (JPEG, PNG, GIF, BMP, TIFF, AVIF), except for WebP. I tested with multiple images, and it only happens with WebP ones.

**To Reproduce**
1. Get a WebP image
2. Try to see its thumbnail
3. Done

**Expected behavior**
Show the thumbnail in File Explorer

**Screenshots**
![image](https://github.com/nomacs/nomacs/assets/32412883/db4b2fb7-7df3-40af-97ab-45547faf76c6)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version IoT Enterprise LTSC 2021 19044.3930

**Comment by novomesk** [comment]
Do you have [Webp Image Extensions](https://apps.microsoft.com/detail/9PG2DK419DRG?hl=en-us&gl=US) from Microsoft Store installed?

**Comment by riomccloud** [comment]
Yep. I tried reinstalling it, and now it worked. Something probably broke. ^^'
